lib/v2/screens/dashboard/components/transactions_list/interactor/viewmodels/member_state.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/member_model.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+class MemberState extends Equatable {
+  final PageState pageState;
+  final String accountName;
+  final MemberModel? member;
+
+  String get displayName => (member != null && member!.nickname != "") ? member!.nickname : accountName;
+  String get profileImageURL => member?.image ?? "";
```

Comment:
why do we need to make this an empty string? 
why not make it nullable? 
in the UI instead of isEmpty we check for null. 

This is the reason we have null safe.

Reply:
It is an empty string. 

We can parse the empty string and make it null - same difference as I am concerned. But it is an empty string as we parse it, so I don't see any reason to change that.